ragna/deploy/_ui/modal_configuration.py

```diff
@@ -146,22 +197,20 @@ async def did_finish_upload(self, uploaded_documents):
             self.start_chat_button.disabled = False
 
     def change_upload_files_label(self, mode="normal"):
+        self.error = False
```

Comment:
Required to introduce `error` so that we can watch it, and update the CSS whenever a user performs an invalid operation.

This was previously implicitly watched by other components, but now that we have added many more components to the modal, we need to introduce an explicit component to watch it (otherwise there are edge cases that arise where the CSS is not updated when a user performs an error).